docs/en/stack/ml/anomaly-detection/ml-population-analysis.asciidoc

```diff
@@ -0,0 +1,96 @@
+[[ml-configuring-populations]]
+= Performing population analysis
+
+Population analysis is a method of detecting anomalies by comparing the behavior of entities or events within a specified population.
+In this approach, {ml} analytics create a profile of what is considered "typical" behavior for users, machines, or other entities over a specified time period.
+An entity is considered as anomalous when its behavior deviates from that of the population, indicating abnormal activity compared to the rest of the population.
+
+This type of analysis is most effective when the behavior within a group is generally homogeneous, allowing for the identification of unusual patterns.
+However, it is less useful when members of the population show vastly different behaviors.
+In such cases, you can segment your data into groups with similar behaviors and run separate jobs for each.
+This can be done by using a query filter in the datafeed or by applying the `partition_field_name` to split the analysis across different groups.
+
+Population analysis is resource-efficient and scales well, enabling the analysis of populations consisting of hundreds of thousands or even millions of entities with a lower resource footprint than analyzing each series individually.
+
+
+
+[discrete]
+[[population-recommendations]]
+== Recommendations
+
+* Use population analysis when the behavior within a group is mostly homogeneous, as it helps identify anomalous patterns effectively.
+* Leverage population analysis when dealing with large-scale datasets.
+* Avoid using population analysis when members of the population exhibit vastly different behaviors, as it may not be effective.
+
+
+[discrete]
+[[creating-population-jobs]]
+== Creating population jobs
+
+. In {kib}, navigate to **{ml-app} > Anomaly Detection > Jobs**.
+. Click **Create {anomaly-jobs}**, select the {data-source} you want to analyze.
```

Comment:
Nit - this ends up displaying as 'Create anomaly detection jobs' in the generated doc, but the button in the UI says 'Create job'.

Reply:
🦅 👁️ 
Great catch, thank you! Updated via [0c5ac80](https://github.com/elastic/stack-docs/pull/2784/commits/0c5ac801006c10659258a63fe161f09212677d8e).